.github/workflows/ci.yaml

```diff
@@ -256,36 +256,18 @@ jobs:
         # NOTE: To speed the CI run, we split unit and integration tests into multiple jobs where
         # each job runs subset of tests.
         include:
-          - name: 'Unit Tests (chunk 1)'
+          - name: 'Unit Tests'
             task: 'ci-unit'
-            nosetests_node_total: 2
-            nosetests_node_index: 0
```

Comment:
Chunking of tests relied on a nosetest plugin. Though pytest might be able to do something similar, we already have this feature in the pants+pytest workflow. As this workflow should be deprecated and removed very soon, I didn't think updating this feature in the Makefile was worth it.

We will, however, have to adjust the required checks before this PR can be merged.

Reply:
The tests took so long to run, so I searched for a pytest plugin and found pytest-shard. I rewrote history to use that instead of removing sharding.